src/loaders/configure/configure.js

```diff
@@ -52,6 +52,7 @@ export function configure (agent, opts = {}, loaderType, forceDrain) {
     ...(updatedInit.ajax.deny_list || []),
     ...(updatedInit.ajax.block_internal ? internalTrafficList : [])
   ]
+  runtime.ptid = agent.agentIdentifier
```

Comment:
Does `ptid` on the RUM or any earlier payloads not mess with anything?

Reply:
This is actually the reason the request was made for us to calculate it, so that PVE's could be tied to a trace by ptid